lib/floki/filter_out.ex

```diff
@@ -49,6 +49,7 @@ defmodule Floki.FilterOut do
 
   defp filter({nodetext, _, _}, selector) when nodetext === selector, do: false
   defp filter({nodetext, _}, selector) when nodetext === selector, do: false
+  defp filter(text, :text) when is_bitstring(text), do: false
```

Comment:
Why not use `is_binary/1` here? Since we are always dealing with `utf8` in text, I think we could consider text as at least a binary. What do you think?

Reply:
You are right, it should be `is_binary` - I will push a fix.